### ISSUE TITLE
Feature request: EE3 compatibility

### ISSUE THREAD

**Issue by ricklecoat** [goodtimeclub/title-commander#1]
I know that the original intention of this GitHub project was to get TitleMaster back on its feet with regard to EE2. But with EE3 now on the scene, is there any chance of this project upgrading its goals to make TitleMaster EE3 compatible? Disclaimer: I've got no PHP skills at all, so I'm unlikely to be of any help in actually achieving such a goal; I'm just a guy who bought a TitleMaster developer licence a couple of months before the devs fell off the planet, and who would love to be able to keep on using the add-on with EE3. So: whaddayathink?

**Comment by goodtimeclub** [comment]
While I do agree with you that it would be great to cater to EE3 as well – frankly, I am not there. In the future, I do plan to update this further but the roadmap is not defined yet.

Title Master definitely fills a gap in some users' workflow so one day I **will** work to ensure compatibility across versions of EE. It's just a matter of when I have time to explore EE3 more and get more comfortable with the API changes(which are, I admit, minimal).

**Comment by ricklecoat** [comment]
Fair enough. I'll be the first to admit that my interest in this is completely self-serving. Not only did I feel a bit hard done by when the original developers abandoned the add-on only a few short months after selling me a developer license (I know, _caveat emptor_, right?), but I have a couple of client sites built using Title Master and not having it in the toolbox any more makes it a less simple prospect to move them up to EE3. And I _like_ the functionality that Title Master provided (it's one of those things that I think should be built into EE).

If and when you consider taking TM through to EE3 compatibility, do let me know. Like I said in my original issue comment, I have no PHP coding skills to speak of, but if you'd like a beta tester then let me know. I realise that an EE3 version is a big ask, so of course I'd like to help out in whatever small way I can.

**Comment by klibansky** [comment]
Wow EE3 would be great!

**Comment by goodtimeclub** [comment]
See https://github.com/goodtimeclub/title-master/issues/2#issuecomment-296719538
